pymc3/distributions/continuous.py

```diff
@@ -173,8 +172,8 @@ class Flat(Continuous):
     """
 
     def __init__(self, *args, **kwargs):
-        super(Flat, self).__init__(*args, **kwargs)
-        self.median = 0
+        self._default = 0
```

Comment:
Ah, I guess we could just override the `default` method. It doesn't look like anything is using the `defaults` attribute directly (except for one call in `transforms.py`, which looks fine). If it had an underscore we'd have an easier time doing stuff like that ;-)